examples/qualcomm/scripts/dummy_llama2.py

```diff
@@ -39,6 +39,9 @@ def create_device_inputs(example_inputs, use_kv_cache):
 
 
 if __name__ == "__main__":
+    print(
```

Comment:
did you run into any issue with the script?

Reply:
I test it last week and it seems ok

Reply:
Hi @cccclai, 

We found some unideal behavior in our CI. For the following reasons we think it's better to have this warning:

1. In 8a8w case, the output shape seems to be different from what it has before.
```bash
 python dummy_llama2.py --ptq 8a8w ...
```
2. In 16a4w case, it even fail to export now.
```bash
 python dummy_llama2.py --ptq 16a4w ...
```
3. Prevent from creating too many issues. bucause users might want to try it, but we are still working on some of its components.

> I test it last week and it seems ok

Would you mind to share your command please? We can also reproduce it and find what the difference. Thanks! :D

Reply:
Ah I take my word back - I just try export the model and see this error when I try to load the model in the runtime

```
[INFO] [Qnn ExecuTorch]: create QNN Logger with log_level 2
[WARNING] [Qnn ExecuTorch]:  <W> Initializing HtpProvider

[WARNING] [Qnn ExecuTorch]:  <W> Function not called, PrepareLib isn't loaded!

[INFO] [Qnn ExecuTorch]: Initialize Qnn backend parameters for Qnn executorch backend type 2
[INFO] [Qnn ExecuTorch]: Caching: Caching is in RESTORE MODE.
[WARNING] [Qnn ExecuTorch]:  <W> sg_stubPtr is not null, skip loadRemoteSymbols


[ERROR] [Qnn ExecuTorch]:  <E> DspTransport.openSession qnn_open failed, 0x80000406

[ERROR] [Qnn ExecuTorch]:  <E> IDspTransport: Unable to load lib 0x80000406

[ERROR] [Qnn ExecuTorch]:  <E> DspTransport failed,cannot open session, error 0x00000009

[ERROR] [Qnn ExecuTorch]:  <E> Unable to load Skel Library. transportStatus: 9

[ERROR] [Qnn ExecuTorch]:  <E> Failed to retrieve skel build id: err: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Failed to create transport for device, error: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Failed to load skel, error: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Transport layer setup failed: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Failed to parse default platform info: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Failed to load default platform info: 1008

[ERROR] [Qnn ExecuTorch]:  <E> Failed to parse platform config: 1008

[ERROR] [Qnn ExecuTorch]: Failed to create device_handle for Backend ID 6, error=1008
E 00:00:00.245462 executorch:QnnManager.cpp:154] Fail to configure Qnn device
E 00:00:00.245471 executorch:QnnExecuTorchBackend.cpp:54] Fail to initialize Qnn Manager
E 00:00:00.245478 executorch:method.cpp:106] Init failed for backend QnnBackend: 0x1
F 00:00:00.245497 executorch:qnn_executor_runner.cpp:215] In function main(), assert failed (method.ok()): Loading of method forward failed with status 0x1
Aborted
```

Any chance you know the reason?

Reply:
Oh also I think the code change in llama_transformer.py might be the culprit when the issue you saw.

Reply:
Actually the error message might be just for me because I only have SM8450. Just open an issue here https://github.com/pytorch/executorch/issues/2788

Reply:
> Oh also I think the code change in llama_transformer.py might be the culprit when the issue you saw.

Thank you for pointing out the possibility. We will investigate it later.

> Actually the error message might be just for me because I only have SM8450. Just open an issue here #2788

We will find a 8450 device and try to reproduce it. Once we have any news we will reply at issue 2788. Thank you for report.

Reply:
Maybe I ask what device you've been using? Is it SM8450?

Reply:
No, I ususally work on SM8550. I don't evevn test a 8450 device personally.